app/lib/core/services/database_provider/db_services.dart

```diff
@@ -0,0 +1,212 @@
+    return remoteConnect;
+  }
+
+  newUser(User newUser) async {
+    var res;
+    networkStatus = await checkConnectivity();
+
+    if (networkStatus == true) {
+      dbConnect = await connectRemote();
+      res = await dbConnect.query('''
+        INSERT INTO transporters(username, password) VALUES (?,?)
+        ''', [newUser.username, newUser.password]);
+      dbConnect.close();
+      return res;
+    } else
+      print("Cannot Register(Connection Error)");
+  }
+
+  Future<dynamic> getUser(User getUser) async {
+    dbConnect = await connectRemote();
+    var userQuery = await dbConnect.query('''
+        SELECT username, password, userID FROM transporters where username = ? AND password = ?
+        ''', [getUser.username, getUser.password]);
+    dbConnect.close();
+    return userQuery;
+  }
+
+  newForm(AnimalTransportRecord newRecord, String withInfo) async {
+    dbConnect = await connectRemote();
+    await dbConnect.query('''
+        INSERT INTO TransporterInfo(companyName, trainingType, companyAddress, addressLastCleanedAt, dateLastCleaned,
+        driversAreBriefed, driversHaveTraining, trailerLicensePlate, trailerProvince, trainingExpiryDate, vehicleLicensePlate,
+        vehicleProvince) VALUES (?,?,?,?,?,?,?,?,?,?,?,?)''', [
+      newRecord.tranInfo.companyName,
+      newRecord.tranInfo.trainingType,
+      newRecord.tranInfo.companyAddress,
+      newRecord.tranInfo.addressLastCleanedAt,
+      newRecord.tranInfo.dateLastCleaned,
+      newRecord.tranInfo.driversAreBriefed,
+      newRecord.tranInfo.driversHaveTraining,
+      newRecord.tranInfo.trailerLicensePlate,
+      newRecord.tranInfo.trailerProvince,
+      newRecord.tranInfo.trainingExpiryDate,
+      newRecord.tranInfo.vehicleLicensePlate,
+      newRecord.tranInfo.vehicleProvince
+    ]);
+
+    await dbConnect.query('''
+        INSERT INTO ShipInfo(departureAddress, LocationId, LocationName, ContactInfo, shipperIsOwner, shipperName) VALUES
+        (?,?,?,?,?,?)''', [
+      newRecord.shipInfo.departureAddress,
+      newRecord.shipInfo.departureLocationId,
+      newRecord.shipInfo.departureLocationName,
+      newRecord.shipInfo.shipperContactInfo,
+      newRecord.shipInfo.shipperIsAnimalOwner,
+      newRecord.shipInfo.shipperName
+    ]);
+
+    await dbConnect.query(
+        '''
+        INSERT INTO ContingencyPlan(communicationPlan, expectedPrepProcess, goalStatement, standardMonitoring) VALUES (?,?,?,?)''',
+        [
+          newRecord.contingencyInfo.communicationPlan,
+          newRecord.contingencyInfo.expectedPrepProcess,
+          newRecord.contingencyInfo.goalStatement,
+          newRecord.contingencyInfo.standardAnimalMonitoring,
+          newRecord.contingencyInfo.potentialSafetyActions(),
+          newRecord.contingencyInfo.contingencyEvents(),
+          newRecord.contingencyInfo.crisisContacts(),
+          newRecord.contingencyInfo.potentialHazards()
+        ]);
+
+    await dbConnect.query('''
```

Comment:
- [ ] Each of these should be their own function, it will help us to narrow down the point of failure in the future if something breaks (like we change an object somewhere, or add a field)